### PR TITLE
fix: use http/1.1 for post for post-processor

### PR DESCRIPTION
**Pull request by chingor13**
Related #5066

### CODE REVIEW

File: packages/owl-bot/cloud-build/update-pr.yaml [hunk 1]
```diff
@@ -97,6 +97,29 @@ steps:
       - 'GITHUB_TOKEN=$_GITHUB_TOKEN'
       - 'HOME=/workspace'
 
+  # TODO: remove when we find a better resolution.
```
Comment (SurferJeffAtGoogle) [inline packages/owl-bot/cloud-build/update-pr.yaml:100]: I think we need a 
```
env:
    - 'HOME=/workspace'
```

On each of these new steps.

File: packages/owl-bot/cloud-build/update-pr.yaml [hunk 2]
```diff
@@ -97,6 +97,29 @@ steps:
...
+      - 'HTTP/1.1'
+  # TODO: remove when we find a better resolution.
+  # See: https://github.com/orgs/community/discussions/55820
+  - name: 'gcr.io/cloud-builders/git'
+    args:
+      - config
+      - --global
+      - http.postBuffer
+      - '157286400'
+  - name: 'gcr.io/cloud-builders/git'
```
Comment (SurferJeffAtGoogle) [inline packages/owl-bot/cloud-build/update-pr.yaml:116]: This looks identical to lines 102-107.

Comment (chingor13) [reply packages/owl-bot/cloud-build/update-pr.yaml:111]: Fixed

File: packages/owl-bot/cloud-build/update-pr.yaml (hunk 1 above)
Comment (chingor13) [reply packages/owl-bot/cloud-build/update-pr.yaml:100]: Fixed

**Review by SurferJeffAtGoogle — APPROVED** [review]

**Pull request merged**